gulpfile.babel.js

```diff
@@ -23,6 +23,7 @@ const nodeResources = (() => {
   return _.chain(result.toString().trim().split(/\r?\n/))
     // remove app root path
     .tap((r) => r.shift())
+    .filter((dep) => dep.match(/node_modules/g).length == 1)
```

Comment:
@dstreet As per [npm3 documetation](https://docs.npmjs.com/how-npm-works/npm3), we can't ignore depth. Some packages may need different versions of common dependency.